packages/core/src/customer/customer-strategy-selector.ts

```diff
@@ -84,6 +92,34 @@ export function createCustomerStrategySelectorFactory(): CustomerStrategySelecto
         },
     );
 
+    const getWalletButtonsStatus = createSelector(
+        (state: CustomerStrategyState) => state,
+        (state) => (methodIds: string[]) => {
+            const walletButtonsStatus: CustomerStrategyStatus = {};
+
+            methodIds.forEach((methodId) => {
+                if (state.data[methodId]) {
+                    walletButtonsStatus[methodId] = {
+                        isInitialized: true,
+                        isLoading: false,
+                    };
+                } else if (state.errors.failedMethodIds?.includes(methodId)) {
+                    walletButtonsStatus[methodId] = {
+                        isInitialized: false,
+                        isLoading: false,
+                    };
```

Comment:
Previouly, we treated a broken wallet button as not initialised, so I'm sticking with that idea here.